docs/tutorial.md

```diff
@@ -343,46 +352,60 @@ Some of the new styles have more than one selector separated by a space. The spa
+When combined with a space, the the selector will match the start button *only* if it is inside a container with a CSS class of "started".
 
-The rule is `"display: none"` which tells Textual to hide the button.
+As before, the `display: none` rule will cause any matching widgets to be hidden from view. 
+
+If we were to write this in English, it would be something like: "Hide the start button if the widget is already started".
 
 ### Manipulating classes
 
 Modifying a widget's CSS classes is a convenient way to update visuals without introducing a lot of messy display related code.
 
-You can add and remove CSS classes with the [add_class()][textual.dom.DOMNode.add_class] and [remove_class()][textual.dom.DOMNode.remove_class] methods. We will use these methods to connect the started state to the Start / Stop buttons.
+You can add and remove CSS classes with the [add_class()][textual.dom.DOMNode.add_class] and [remove_class()][textual.dom.DOMNode.remove_class] methods.
+We will use these methods to connect the started state to the Start / Stop buttons.
 
-The following code will start or stop the stopwatches in response to clicking a button.
+The following code will start or stop the stopwatches in response to clicking a button:
 
 ```python title="stopwatch04.py" hl_lines="13-18"
 --8<-- "docs/examples/tutorial/stopwatch04.py"
 ```
 
-The `on_button_pressed` method is an *event handler*. Event handlers are methods called by Textual in response to an *event* such as a key press, mouse click, etc. Event handlers begin with `on_` followed by the name of the event they will handle. Hence `on_button_pressed` will handle the button pressed event.
+The `on_button_pressed` method is an *event handler*. Event handlers are methods called by Textual in response to an *event* such as a key press, mouse click, etc.
+Event handlers begin with `on_` followed by the name of the event they will handle.
+Hence `on_button_pressed` will handle the button pressed event.
 
 If you run `stopwatch04.py` now you will be able to toggle between the two states by clicking the first button:
 
 ```{.textual path="docs/examples/tutorial/stopwatch04.py" title="stopwatch04.py" press="tab,tab,tab,enter"}
 ```
 
+When the button event handler adds or removes the `"started"` CSS class, Textual re-applies the CSS and updates the visuals.
+
+
 ## Reactive attributes
 
-A recurring theme in Textual is that you rarely need to explicitly update a widget. It is possible: you can call [refresh()][textual.widget.Widget.refresh] to display new data. However, Textual prefers to do this automatically via _reactive_ attributes.
+A recurring theme in Textual is that you rarely need to explicitly update a widget's visuals.
+It is possible: you can call [refresh()][textual.widget.Widget.refresh] to display new data.
+However, Textual prefers to do this automatically via _reactive_ attributes.
```

Comment:
Feels like we're describing how to declare and giving an example of using "reactive" before defining what it actually _is_. It's tricky to follow examples that contain a new concept before you've been given at least a high-level/simplified definition.

Maybe add something like (probably improve the wording):

```suggestion
It is possible: you can call [refresh()][textual.widget.Widget.refresh] to display new data.
However, Textual prefers to do this automatically via _reactive_ attributes. A reactive attribute is an attribute that, when set, will automatically refresh the widget.
```